docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -261,9 +257,25 @@ applications.
 | `http://tizen.org/privilege/volume.set`  | public   | -            | 2.3.1   | The application can adjust the volume for different features, such as notification alerts, ringtones, and media. |
 | `http://tizen.org/privilege/widget.viewer` | public   | -            | 2.3.1 | The application can show widgets, and information from their associated applications, on the home screen. |
 | `http://tizen.org/privilege/window.priority.set` | public   | -            | 2.3.1   | The application can appear on top of other windows and screens, including the lock screen, according to the order of priority of the windows. This can prevent the user from interacting with other applications or screens until the window for the application is closed. |
-| `http://tizen.org/privilege/zigbee`      | public   | -            | 4.0   | The application can connect a ZigBee coordinator to end devices and control connected end devices. |
-| `http://tizen.org/privilege/zigbee.admin`      | platform   | -            | 4.0   | The application can control a connected ZigBee coordinator, e.g. turning it on or off. |
 
+<a name="IoT"></a>
+## IoT Native API Privileges
+
+The following table lists the API privileges, which you must declare
+when using security-sensitive API modules in IoT native
+applications.
+
```

Comment:
Hello, @sabeena21 
There is another PR which is modifying docs/application/native/tutorials/details/sec-privileges.md file, too.
The PR https://github.com/Samsung/tizen-docs/pull/372 changes the overall,
so changing the period to the entire document in this PR can cause a merge conflict.

Please refer https://github.com/Samsung/tizen-docs/pull/372.

As you can see on the #372, to prevent the merge conflict,
I think it would be better to revert the changing period with colon for line 52, 164, and 290.

In this PR,
please focus on the subject of this PR, creating IoT native API privileges table only,
and don't modify other things.

Reply:
@mijin85cho Okay. I just reverted changed periods. Please confirm that.